Conversion to v11 and added evaluation content/plan (#3)
Open-Systems-Pharmacology/Rosuvastatin-Model: Evaluation/Input/Content/Section1_Introduction.md

> Rosuvastatin is an inhibitor of the liver 3-hydroxy-3-methyl-glutaryl-coenzyme A reductase (HMG-CoA reductase), and it is prescribed to treat hyperlipidemia. Rosuvastatin is metaboliszed by CYP2C9 to a small extent, transported by OATP1B1 and OATP1B3 into the liver and subsequently excreted into the bile by BCRP and highly renally excreted suggesting active tubular secretion (by OATP2B1, OAT3 and Pgp). The absolut bioavailability of rosuvastatin is about 20% mainly due to poor absorption and the fraction excreted in feces and urine after an oral dose is about 90% and 10% respectively, both mainly as parent compound. Rosuvastatin is a victim of OATP1B1, OATPB1B3 and BCRP and can therefore serve as a victim for these transporters in DDI studies. 

Reviewer: ```suggestion
Rosuvastatin is an inhibitor of the liver 3-hydroxy-3-methyl-glutaryl-coenzyme A reductase (HMG-CoA reductase) and is prescribed to treat hyperlipidemia. Rosuvastatin is metabolized by CYP2C9 to a small extent, transported by OATP1B1 and OATP1B3 into the liver, and subsequently excreted into the bile by BCRP. The renal excretion of rosuvastatin is high, suggesting active tubular secretion (by the transporters OATP2B1, OAT3, and Pgp). The absolute bioavailability of rosuvastatin is about 20%, mainly due to poor absorption. The fraction excreted in feces and urine after an oral dose is about 90% and 10%, respectively, both mainly as parent compound. Rosuvastatin is a substrate of OATP1B1, OATPB1B3, and BCRP and can, therefore, serve as a victim for these transporters in DDI studies. 
```

> The herein presented PBPK model of rosuvastatin has been developed using published pharmacokinetic clinical data by Billington et al. 2019 ([Billington 2019](#5-references)), Martin et al. 2003 ([Martin 2003c](#5-references)), Stopfer et al. 2016 ([Stopfer 2016](#5-references)), Stopfer et al. 2018 ([Stopfer 2018b](#5-references)), Wu et al. 2017 ([Wu 2017](#5-references)), Gidal et al. 2017 ([Gidal  2017])(#5-references), Cooper et al. 2003 ([Cooper 2003a](#5-references)),  Lee et al. 2018 ([Lee 2018](#5-references)) and Gosai et al. 2008 ([Gosai 2008](#5-references)). 

Reviewer: ```suggestion
The herein presented PBPK model of rosuvastatin has been developed using published pharmacokinetic clinical data by Billington et al. 2019 ([Billington 2019](#5-references)), Martin et al. 2003 ([Martin 2003c](#5-references)), Stopfer et al. 2016 ([Stopfer 2016](#5-references)), Stopfer et al. 2018 ([Stopfer 2018b](#5-references)), Wu et al. 2017 ([Wu 2017](#5-references)), Gidal et al. 2017 ([Gidal  2017])(#5-references)), Cooper et al. 2003 ([Cooper 2003a](#5-references)),  Lee et al. 2018 ([Lee 2018](#5-references)) and Gosai et al. 2008 ([Gosai 2008](#5-references)). 
```

The model has then been evaluated by comparing observed data to simulations of both intravenously and orally administered rosuvastatin covering a dose range of 0.002 mg to 80.0 mg. Both single dose and multiple dose administration are represented in the development and evaluation data sets. 

The presented model includes the following features:

- metabolism by CYP2C9,
- transport by OATP1B1 and OATP1B3,
- transport by BCRP,
- transport by OATP2B1,
- transport by Pgp,
- transport by OAT3,
- renal clearance by glomerular filtration,

> - oral absorption with dissolution rate assigned to the Weibull function.

Reviewer: ```suggestion
- oral absorption with dissolution rate described by a Weibull function.
```
